### PR TITLE
Python to production

### PR DESCRIPTION
**Pull request by rcehemann**
Added stress-test in py_feels/binary/BC_stress_test.py. binary_classifier.py now classifies the last tweet added to 'tweets' table and appends 'feels' table with sentiment data. Updated setup script to install required SQL libraries.

### CODE REVIEW

File: py_feels/binary/binary_classifier.py [hunk 1]
```diff
@@ -12,25 +12,31 @@
...
 
-# read amazon training data for testing loaded model
-amazon = pd.read_csv(
-        './training/UCI_training_data/amazon_cells_labelled.txt', 
-        sep="\t", 
-        names=['sentence', 'sentiment'], 
-        dtype={'sentence':str, 'sentiment':int}
-)
+# connect to  SQL database, query for last item in 'tweets' table
+engine = create_engine('postgresql+psycopg2://wojak:@localhost/bit_feels_dev')
```
Comment (thekeele) [inline py_feels/binary/binary_classifier.py]: The database will change based on the environment. The dev db is `bit_feels_dev` and the prod db is `bit_feels`. Generally the environment is supplied when the app runs, something like `PY_ENV=prod binary_classifier.py`

File: py_feels/binary/binary_classifier.py [hunk 2]
```diff
@@ -12,25 +12,31 @@
...
+# construct feels DataFrame for passing back to SQL
+feels = pd.DataFrame()
+feels['sentiment']   = sentiments
+feels['classifier']  = dict(model.steps)['clf'].__class__.__name__
+feels['tweet_id']    = tweets.tweet_id.values
 
+# store prediction time
+time_now = str(datetime.datetime.now())
+feels['inserted_at'] = time_now
+feels['updated_at']  = time_now
```
Comment (thekeele) [inline py_feels/binary/binary_classifier.py]: Can this be `nil`?

File: py_feels/binary/binary_classifier.py [hunk 3]
```diff
@@ -12,25 +12,31 @@
...
-# read amazon training data for testing loaded model
-amazon = pd.read_csv(
-        './training/UCI_training_data/amazon_cells_labelled.txt', 
-        sep="\t", 
-        names=['sentence', 'sentiment'], 
-        dtype={'sentence':str, 'sentiment':int}
-)
+# connect to  SQL database, query for last item in 'tweets' table
+engine = create_engine('postgresql+psycopg2://wojak:@localhost/bit_feels_dev')
+query  = "SELECT * FROM tweets WHERE id=(SELECT max(id) FROM tweets)"
```
Comment (thekeele) [inline py_feels/binary/binary_classifier.py]: Multiple `SELECT` should be avoided. Using `id` would work but we should probably use the timestamps. 

This should work:
`select text from tweets order by inserted_at desc limit 1;`

File: py_feels/binary/binary_classifier.py (hunk 2 above)
Comment (rcehemann) [reply py_feels/binary/binary_classifier.py]: it comes from the system time, so I don't think so

**Pull request merged**